src/device-manager/WeaveDataManagementClient.cpp

```diff
@@ -1063,6 +1267,29 @@ void WdmClient::ClientEventCallback(void * const aAppState, SubscriptionClient::
         pWdmClient->ClearOpState();
         break;
 
+    case SubscriptionClient::kEvent_OnEventStreamReceived:
+        WeaveLogDetail(DataManagement, "kEvent_OnEventStreamReceived");
+
+        if (pWdmClient->mWdmEventProcessor != nullptr)
```

Comment:
style, Use NULL in weave

Reply:
For unique_ptr, nullptr instead of NULL should be used.

Reply:
ack, thanks